e2e/cli_test.go

```diff
@@ -0,0 +1,206 @@
+			firstLine:   "Delete a Realm app",
+		},
+		{
+			description: "the app diff command",
+			args:        []string{"app", "diff"},
+			firstLine:   "Show differences between your local directory and your Realm app",
+		},
+		{
+			description: "the app describe command",
+			args:        []string{"app", "describe"},
+			firstLine:   "Displays information about your Realm app",
+		},
+		{
+			description: "the user create command",
+			args:        []string{"user", "create"},
+			firstLine:   "Create an application user for your Realm app",
+		},
+		{
+			description: "the user list command",
+			args:        []string{"user", "list"},
+			firstLine:   "List the application users of your Realm app",
+		},
+		{
+			description: "the user disable command",
+			args:        []string{"user", "disable"},
+			firstLine:   "Disable an application User of your Realm app",
+		},
+		{
+			description: "the user enable command",
+			args:        []string{"user", "enable"},
+			firstLine:   "Enable an application User of your Realm app",
+		},
+		{
+			description: "the user revoke command",
+			args:        []string{"user", "revoke"},
+			firstLine:   "Revoke an application User’s sessions from your Realm app",
+		},
+		{
+			description: "the user delete command",
+			args:        []string{"user", "delete"},
+			firstLine:   "Delete an application user from your Realm app",
+		},
+		{
+			description: "the secrets create command",
+			args:        []string{"secrets", "create"},
+			firstLine:   "Create a Secret for your Realm app",
+		},
+		{
+			description: "the secrets list command",
+			args:        []string{"secrets", "list"},
+			firstLine:   "List the Secrets in your Realm app",
+		},
+		{
+			description: "the secrets update command",
+			args:        []string{"secrets", "update"},
+			firstLine:   "Update a Secret in your Realm app",
+		},
+		{
+			description: "the secrets delete command",
+			args:        []string{"secrets", "delete"},
+			firstLine:   "Delete a Secret from your Realm app",
+		},
+		{
+			description: "the function run command",
+			args:        []string{"function", "run"},
+			firstLine:   "Run a Function from your Realm app",
+		},
+		{
+			description: "the logs list command",
+			args:        []string{"logs", "list"},
+			firstLine:   "Lists the Logs in your Realm app",
+		},
+		{
+			description: "the schema datamodels command",
+			args:        []string{"schema", "datamodels"},
+			firstLine:   "Generate data models based on your Schema",
+		},
+		{
+			description: "the accesslist create command",
+			args:        []string{"accesslist", "create"},
+			firstLine:   "Create an IP address or CIDR block in the Access List for your Realm app",
+		},
+		{
+			description: "the accesslist list command",
+			args:        []string{"accesslist", "list"},
+			firstLine:   "List the allowed entries in the Access List of your Realm app",
+		},
+		{
+			description: "the accesslist list command",
```

Comment:
should this be "the accesslist **update** command"?

Reply:
it most certainly should, thanks for catching!